.github/workflows/playwright.yml

```diff
@@ -1,22 +1,53 @@
 name: Playwright Tests
+
 on:
   workflow_call:
     inputs:
       frontendHostname:
         type: string
         description: 'Hostname of the frontend server (optional)'
         required: false
+      playwright_branch:  
+        type: string
+        description: 'The name of the branch to be tested with Playwright.'
+        required: false  
   workflow_dispatch:
     inputs:
       frontendHostname:
         type: string
         description: 'Hostname of the frontend server (optional)'
-        required: false
+        required: true
+      playwright_branch:  
+        type: string
+        description: 'The name of the branch to be tested with Playwright.'
+        required: true 
   push:
     branches: [ main ]
 
 jobs:
+
+  determine_playwright_branch:
```

Comment:
is this needed? I think this can go, and we can set a default in the variables above to main. Then the ref: can take the input from the variable

Reply:
I'm including the 'determine_playwright_branch' Job in the schulportal-testautomatisierung. In this way we won't need to duplicate this Job in the spsh app repositories

Reply:
but the exact same code is already part of the spsh_app_deploy, i think that that is enough